go-controller/pkg/ovn/pods_test.go

```diff
@@ -805,6 +805,8 @@ var _ = ginkgo.Describe("OVN Pod Operations", func() {
 
 				defer cancel()
 				resetNBClient(connCtx, fakeOvn.controller.nbClient)
+				// reset backoff for immediate retry
+				fakeOvn.controller.retryPods.setRetryObjWithNoBackoff(key)
```

Comment:
Should we trigger an immediate retry also for all the times we call `requestRetryObjs` in pods_test.go, to expedite unit tests?

Reply:
oh no, I simply saw that in a few cases, like this one, you set the backoff to 0 before calling `requestRetryObjs`, thus triggering an immediate retry, and I was wondering if we should do the same to all the other cases in unit tests where we call `requestRetryObjs()`. The previous implementation of `requestRetryObjs` implied an immediate retry (with `updateAll=true`) and now that the boolean is gone we just base our decision on the object backoff value.

Reply:
oh, let me double check those cases. I think I only fixed these cause thats where the tests failed :)

Reply:
there were a few more cases where we needed to request immediate retry. Fixed those.